src/compiler/emitter.ts

```diff
@@ -1084,16 +1084,9 @@ const _super = (function (geti, seti) {
                 return;
             }
 
-            const indentBeforeDot = needsIndentation(node, node.expression, node.dotToken);
-            const indentAfterDot = needsIndentation(node, node.dotToken, node.name);
-            const shouldEmitDotDot = !indentBeforeDot && needsDotDotForPropertyAccess(node.expression);
-
             emitExpression(node.expression);
-            increaseIndentIf(indentBeforeDot);
-            write(shouldEmitDotDot ? ".." : ".");
-            increaseIndentIf(indentAfterDot);
+            write(needsDotDotForPropertyAccess(node.expression) ? ".." : ".");

```

Comment:
Per [this comment](https://github.com/Microsoft/TypeScript/pull/9294/commits/9abe0f8953216b204fb0a3d4d09b7e532ef7fe26#r69052612), we need to infer the indentation as if we had parsed the `DotToken`.

This is what we do in master:
```ts
const dotRangeStart = nodeIsSynthesized(node.expression) ? -1 : node.expression.end;
const dotRangeEnd = nodeIsSynthesized(node.expression) ? -1 : skipTrivia(currentText, node.expression.end) + 1;
const dotToken = <TextRange>{ pos: dotRangeStart, end: dotRangeEnd };
const indentedBeforeDot = indentIfOnDifferentLines(node, node.expression, dotToken);
```

We could probably do something like this:

```ts
const dotToken = <Node>{ kind: SyntaxKind.DotToken, pos: node.expression.end, end: skipTrivia(currentText, node.expression.end) + 1 };
const indentBeforeDot = needsIndentation(node, node.expression, dotToken);
const indentAfterDot = needsIndentation(node, dotToken, node.name);
const shouldEmitDotDot = !indentBeforeDot && needsDotDotBeforePropertyAccess(node.expression);

// restore the previous increase/decrease indent lines.
```